### PR TITLE
Remove the path tag from the default server views

### PR DESCRIPTION
**Pull request by ramonza**
Fixes: #111

### CODE REVIEW

File: stats/HTTP.md [hunk 1]
```diff
@@ -88,13 +87,23 @@ starts and are available in the context for the entire incoming request processi
 `http_server_status` is set when an incoming request finishes and is only available around the stats
 recorded at the end of request processing.
 
+`http_server_path` and `http_server_host` are set by the client: if the client is untrused (e.g. public-facing servers)
```
[the remark this answers is not in the document]
Comment (ramonza) [reply stats/HTTP.md]: Updated the wording to not be specific to untrusted clients.

**Comment by ramonza** [comment]
@bogdandrutu PTAL

File: stats/HTTP.md [hunk 2]
```diff
@@ -88,13 +87,23 @@ starts and are available in the context for the entire incoming request processi
 `http_server_status` is set when an incoming request finishes and is only available around the stats
 recorded at the end of request processing.
 
+`http_server_path` and `http_server_host` are set by the client: you should be careful about using these
```
Comment (bogdandrutu) [inline stats/HTTP.md]: Maybe same comment and changes should be added to the client side stats.

Comment (ramonza) [reply stats/HTTP.md]: Removed path from HTTP client views as well

**Pull request merged**